qfieldsync/gui/cloud_projects_dialog.py

```diff
@@ -123,18 +125,20 @@ def __init__(
                 parent=self,
             )
         else:
-            self.welcomeLabelValue.setText(
-                self.network_manager.auth().config("username")
+            self.welcomeLabel.setText(
+                self.tr("Greetings {}.").format(
```

Comment:
I would love to keep the username in bold.

Reply:
I was aiming at keeping bold for subsections. If you're passionate about the bold here I can re-add. I'm -0, if you're +1 by my count you win ;)

Reply:
I would love to see the avatar on the top right, something to add in our respective "if we ever get free time" list ;)

Reply:
keep it like this then, let's see how my free saturday time goes :)